### PR TITLE
stats page

### LINKED ISSUE #84: [Frontend] Stats Page

### LINKED ISSUE #87: [Frontend] add latest contributions to Home page

### PR DESCRIPTION
**Pull request by CJ-Rose**

### CODE REVIEW

**Comment by CJ-Rose** [comment]
@kittybest i have moved the `Welcome` component with the link to start the server into the home page, so once the link is opened the rest of the pages should be connecting correctly. but i still haven't figured out how to connect to the server without it 🫤
also have removed much of the content from `Contribute` page to use in `Stats` page, so you can go ahead and merge this into ceremony-ui branch if this works for you.

**Comment by kittybest** [comment]
I think the routes in frontend look a little bit weird.
截圖 2023-08-17 下午2 43 23
截圖 2023-08-17 下午2 43 34
截圖 2023-08-17 下午2 43 42

File: packages/frontend/src/pages/Home.jsx [hunk 1]
```diff
@@ -3,10 +3,17 @@ import { Link } from 'react-router-dom'
...
+import ContributionCard from '../components/ContributionCard'
 import FaqDropdown from '../components/FaqDropdown'
 import Footer from '../components/Footer'
+import state from '../contexts/state'
 
 export default observer(() => {
+  const { ui, ceremony } = React.useContext(state)
+  if (!ceremony.HTTP_SERVER) {
+    return <Welcome />
+  }
```
Comment (kittybest) [inline packages/frontend/src/pages/Home.jsx]: I think I found the reason why my routes seem so weird.
Why here return `<Welcome />` page?

**Comment by kittybest** [comment]
> @kittybest i have moved the `Welcome` component with the link to start the server into the home page, so once the link is opened the rest of the pages should be connecting correctly. but i still haven't figured out how to connect to the server without it 🫤 also have removed much of the content from `Contribute` page to use in `Stats` page, so you can go ahead and merge this into ceremony-ui branch if this works for you.

I finally understand what do you mean by commenting my questions 😂 just removed them. Let me try if we could connect to the server without the `Welcome` component. If I have any update, will directly make a PR to this PR.

**Comment by CJ-Rose** [comment]
@kittybest can you check failing tests please? i'm not sure why they would pass in your PR #89 but not pass here. i didn't change anything, just merged.

**Comment by kittybest** [comment]
> @kittybest can you check failing tests please? i'm not sure why they would pass in your PR #89 but not pass here. i didn't change anything, just merged.

I re-run it and they have passed :)

**Comment by vivianjeng** [comment]
@CJ-Rose I changed the webpack config for vercel [1ca1f97](https://github.com/Unirep/trusted-setup/pull/86/commits/1ca1f97b5aa20160d220872d6574c5d938b69fe4)
please pull the latest commitment
and the latest deployment is here: https://trusted-setup-git-stats-page-unirep.vercel.app/

**Pull request merged**